src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/VersionPageTests.kt

```diff
@@ -136,21 +136,15 @@ class VersionPageTests : TeamcityTests()
     @Test
     fun `renders report tab correctly`()
     {
-        val xmlResponse = template.xmlResponseFor(testModel)
-
-        val xPathRoot = "//div[@id='report-tab']"
-
-        assertThat(xmlResponse, hasXPath("$xPathRoot/h1/text()",
-                equalToCompressingWhiteSpace("r1 display")))
-        assertThat(xmlResponse, hasXPath("$xPathRoot/p[1]/text()",
-                equalToCompressingWhiteSpace("r1-v1")))
-
-        assertThat(xmlResponse, hasXPath("$xPathRoot/p[@id='param-values']", equalToCompressingWhiteSpace("Parameter values: p1=v1, p2=v2")))
-
-        assertThat(xmlResponse, hasXPath("$xPathRoot/iframe/@src", equalTo("/testFocalArtefactUrl")))
-        assertThat(xmlResponse, hasXPath("$xPathRoot/div[@class='text-right']/a/text()",
-                equalToCompressingWhiteSpace("View fullscreen")))
-        assertThat(xmlResponse, hasXPath("$xPathRoot/div[@class='text-right']/a/@href", equalTo("/testFocalArtefactUrl")))
+        val jsoupDoc = template.jsoupDocFor(testModel)
```

Comment:
For reasons which aren't clear to me, a couple of these tests were failing (complaining about a non-existent class attribute on the report-tags element), but passed when I converted them from xPath to jsoup...